### PR TITLE
final commit for Game of Life

### PR DESCRIPTION
**Pull request by BLehrer99**

### CODE REVIEW

**Comment by Mr-Darfler** [comment]
@awualex please add your review to Ben's Code

**Review by Jkohl1 — APPROVED** [review]
Looks good

**Review by awualex — APPROVED** [review]
Just personal thought. Maybe you could create a new button(entered with values) to randomize the initial cell state. And then you could implement the mouse click if statement to switch the cell state at the position of mouse x and mouse y. it's just quite tedious to click all the cells one by one especially when the canvass is big

File: Class-Code/Game of Life/blehrer/sketch.js [hunk 1]
```diff
@@ -1,14 +1,150 @@
...
+const cellCount = 100;
+
+let started = false;
+let startButton;
+
+let paused = false;
+let pauseButton;
+
+let resetButton;
+
```
Comment (Mr-Darfler) [inline Class-Code/Game of Life/blehrer/sketch.js:22]: I appreciate how these variables are arranged

File: Class-Code/Game of Life/blehrer/sketch.js [hunk 2]
```diff
@@ -1,14 +1,150 @@
...
+    for (let i = 0; i < cellCount; ++i) {
+        for (let j = 0; j < cellCount; ++j) {
+            if (ca[i][j] == 1)
+                return false;
+        }
+    }
+    return true;
+}
+
+function initCA() {
```
Comment (Mr-Darfler) [inline Class-Code/Game of Life/blehrer/sketch.js:103]: Does this need its own function? What's your thinking here. Just curious

**Review by Mr-Darfler — APPROVED** [review]

**Pull request merged**